docs/readthedocs/config.md

```diff
@@ -310,6 +310,9 @@ are described in the documentation of [voltage limits](models/voltage_limits).
 
 ## Power flow configuration 
 
+The powerflow configuration in SIMONA is an optional configuration. If no configuration is present, SIMONA will skip all
```

Comment:
may we add a comment in 'vn_simona.conf' for this?

Reply:
I added a coment in config. But I would like to keep this info also in the documentation.